curriculum/test/utils/sort-challenges.test.js

```diff
@@ -0,0 +1,252 @@
+    challengeOrder: 1
+  },
+  {
+    name: 'HTML - project 2 - step 2',
+    superOrder: 1,
+    order: 2,
+    challengeOrder: 2
+  },
+  {
+    name: 'HTML - project 2 - step 3',
+    superOrder: 1,
+    order: 2,
+    challengeOrder: 3
+  },
+  {
+    name: 'HTML - project 2 - step 4',
+    superOrder: 1,
+    order: 2,
+    challengeOrder: 4
+  },
+  {
+    name: 'HTML - project 3 - step 3',
+    superOrder: 1,
+    order: 3,
+    challengeOrder: 1
+  },
+  {
+    name: 'HTML - project 3 - step 2',
+    superOrder: 1,
+    order: 3,
+    challengeOrder: 2
+  },
+  {
+    name: 'HTML - project 3 - step 3',
+    superOrder: 1,
+    order: 3,
+    challengeOrder: 3
+  },
+  {
+    name: 'HTML - project 3 - step 4',
+    superOrder: 1,
+    order: 3,
+    challengeOrder: 4
+  },
+  {
+    name: 'CSS - project 1 - step 1',
+    superOrder: 2,
+    order: 1,
+    challengeOrder: 1
+  },
+  {
+    name: 'CSS - project 1 - step 2',
+    superOrder: 2,
+    order: 1,
+    challengeOrder: 2
+  },
+  {
+    name: 'CSS - project 1 - step 3',
+    superOrder: 2,
+    order: 1,
+    challengeOrder: 3
+  },
+  {
+    name: 'CSS - project 1 - step 4',
+    superOrder: 2,
+    order: 1,
+    challengeOrder: 4
+  },
+  {
+    name: 'CSS - project 2 - step 1',
+    superOrder: 2,
+    order: 2,
+    challengeOrder: 1
+  },
+  {
+    name: 'CSS - project 2 - step 2',
+    superOrder: 2,
+    order: 2,
+    challengeOrder: 2
+  },
+  {
+    name: 'CSS - project 2 - step 3',
+    superOrder: 2,
+    order: 2,
+    challengeOrder: 3
+  },
+  {
+    name: 'CSS - project 2 - step 4',
+    superOrder: 2,
+    order: 2,
+    challengeOrder: 4
+  },
+  {
+    name: 'CSS - project 3 - step 3',
+    superOrder: 2,
+    order: 3,
+    challengeOrder: 1
+  },
+  {
+    name: 'CSS - project 3 - step 2',
+    superOrder: 2,
+    order: 3,
+    challengeOrder: 2
+  },
+  {
+    name: 'CSS - project 3 - step 3',
+    superOrder: 2,
+    order: 3,
+    challengeOrder: 3
+  },
+  {
+    name: 'CSS - project 3 - step 4',
+    superOrder: 2,
+    order: 3,
+    challengeOrder: 4
+  },
+  {
+    name: 'JS - project 1 - step 1',
+    superOrder: 3,
+    order: 1,
+    challengeOrder: 1
+  },
+  {
+    name: 'JS - project 1 - step 2',
+    superOrder: 3,
+    order: 1,
+    challengeOrder: 2
+  },
+  {
+    name: 'JS - project 1 - step 3',
+    superOrder: 3,
+    order: 1,
+    challengeOrder: 3
+  },
+  {
+    name: 'JS - project 1 - step 4',
+    superOrder: 3,
+    order: 1,
+    challengeOrder: 4
+  },
+  {
+    name: 'JS - project 2 - step 1',
+    superOrder: 3,
+    order: 2,
+    challengeOrder: 1
+  },
+  {
+    name: 'JS - project 2 - step 2',
+    superOrder: 3,
+    order: 2,
+    challengeOrder: 2
+  },
+  {
+    name: 'JS - project 2 - step 3',
+    superOrder: 3,
+    order: 2,
+    challengeOrder: 3
+  },
+  {
+    name: 'JS - project 2 - step 4',
+    superOrder: 3,
+    order: 2,
+    challengeOrder: 4
+  },
+  {
+    name: 'JS - project 3 - step 3',
```

Comment:
Similar problem as before:
```suggestion
    name: 'JS - project 3 - step 1',
```